Libraries/D3D12RaytracingFallback/src/TreeletReorder.hlsl

```diff
@@ -229,8 +229,8 @@ void main( uint3 DTid : SV_DispatchThreadID )
 
                 hierarchyBuffer[partition.NodeIndex].LeftChildIndex = leftEntry.NodeIndex;
                 hierarchyBuffer[partition.NodeIndex].RightChildIndex = rightEntry.NodeIndex;
-                hierarchyBuffer[leftEntry.NodeIndex].ParentIndex = partition.NodeIndex;
-                hierarchyBuffer[rightEntry.NodeIndex].ParentIndex = partition.NodeIndex;
+                hierarchyBuffer[leftEntry.NodeIndex].ParentIndex = partition.NodeIndex;                
```

Comment:
I don't really care, but looks like this change is only coming up because of spaces